tool/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
To confirm, this is needed for eclipse?  IntelliJ works off of pom.xml

Reply:
Removed them